augurface/vue.config.js

```diff
@@ -1,4 +1,4 @@
 // #SPDX-License-Identifier: MIT
 module.exports = {
```

Comment:
@mscherer : This change makes me wonder if you are running `augurface` with the intention of having it be Augur's primary frontend, or if you are using it to load repos (which is what it is for, and its not quite right/perfect yet.).  The main frontend is in the `frontend` directory. 

There is also another project, https://github.com/chaoss/augur_view that we are developing new frontend "stuff" in.